config/test.json

```diff
@@ -25,7 +25,7 @@
 		"ad-tracking": false,
 		"a4a-dev-sites": true,
 		"akismet/checkout-quantity-dropdown": true,
-		"automated-migration/application-password": false,
+		"automated-migration/application-password": true,
```

Comment:
why is this being toggled here?

Reply:
This toggles the feature flag enabled on unity tests environment, when running `yarn test-client` .
When I first introduced the feature flag PR, I was conservative and set it false until we actually have some feature to test.